README.md

```diff
@@ -61,36 +63,51 @@ core library since it was quite good already.
 
 Check `example-usage` folder for a basic usage of the library in both TS and JS. But mostly, all you
 need is to add it as a dependency, create a new client and use as normally :)
+Also, you can check `README_DIST.md` for the README present on the npm package, example usage should be
+explicit there so not repeating it here.
 
 ## Build
 
 For a simple command that does all the building of the current source code inside docker as done on CI:
 `docker-compose -f docker-compose-ci.yml run build_from_schema`
 
-If you want to update from the remote schema API, you can do:
+Keep in mind the previous command does not incorporate any changes done to the `base_client` file.
+If you also want to build the final client (useful if changes were made in `base_client`) update from the
+remote schema API, you can do:
+`docker-compose -f docker-compose-ci.yml run rebuild_from_schema`
+
+If you want to update the current code based on the current remote openAPI schema, you can do:
```

Comment:
Is `remote` schema the official schema at https://docs.lune.co/openapi.yml?

Reply:
If so, I don't think the term remote is appropriate, because it's the default schema.

Reply:
Well, it's still the `remote` schema. It just so happens it's also the default. I'll reword things here though, it actually reads a bit confusing. How about:
`If you want to rebuild the library based on the official openAPI schema (https://docs.lune.co/openapi.yml), you can do:`